### DISCUSSION TITLE
babeljs.io repl doesn't seem to be polyfilling correctly

### DISCUSSION THREAD

**Discussion by jbryson3** [babel/babel#13388] [Q&A]
I used babeljs.io "Try it out" feature to get a look at what the ie6 equivalent code would look like. It doesn't seem to be polyfilling anything including a sample `Object.assign` call.

https://babeljs.io/repl#?browsers=ie%206&build=&builtIns=false&corejs=false&spec=false&loose=false&code_lz=DYUwLgBAhhC8EG8BQBIKAjAxgLggRgCYBmAGiQghQBMQAzXAcloHtmGkBfAbiSQHl0AKxCYwAOigBnSQEsA5gDsAFAg4lEYEJLDZCRNdACUQA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=true&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=env&prettier=false&targets=&version=7.14.3&externalPlugins=

This may be due to it being unable to load the polyfill plugin.
<img width="645" alt="image" src="https://user-images.githubusercontent.com/675098/120018644-77832000-bfb5-11eb-8b5d-a486c41bf042.png">

Can anyone else confirm that this is an issue?

**Comment by nicolo-ribaudo** [comment]
It seems like a `bundle.run` or Cloudflare bug: the polyfill CDN returns an error (https://bundle.run/@babel/polyfill@7).
